model/model.go

```diff
@@ -235,6 +235,8 @@ type TableInfo struct {
 	ShardRowIDBits uint64
 	// MaxShardRowIDBits uses to record the max ShardRowIDBits be used so far.
 	MaxShardRowIDBits uint64 `json:"max_shard_row_id_bits"`
+	// AutoRandomBits is used to set the bit number to shard automatically when PKIsHandle.
+	AutoRandomBits uint64 `json:"auto_shard_bits"`
```

Comment:
@tangenta Missed this when reviewing. Is it intended that the JSON field be called `auto_shard_bits` instead of `auto_random_bits`?

Reply:
It is a mistake. I fixed it in #702.